client/app/cloud/project/storage/storage-add/storage-add.html

```diff
@@ -1,39 +1,37 @@
-<div class="cui-page">
+<section class="cui-page__content">
 
-    <div data-ng-include="'app/cloud/project/storage/storage-header.html'"></div>
+    <cui-message-container data-messages="messages"></cui-message-container>
 
-    <section class="cui-page__content">
-        <oui-back-button data-title="{{ :: contentTitle }}"></oui-back-button>
+    <oui-back-button data-title="{{ :: contentTitle }}"></oui-back-button>
 
-        <div data-ng-show="!loaders.init">
+    <div data-ng-show="!loaders.init">
 
-            <form role="form" name="addForm" novalidate="novalidate">
+        <form role="form" name="addForm" novalidate="novalidate">
 
-                <div class="row">
+            <div class="row">
 
-                    <div class="col-md-6 col-lg-4 center add-button-item"
-                         data-ng-repeat="item in items.value"
-                         data-ng-class="{
-                            'active': addedType.value == item
-                         }">
-
-                        <div class="form-group add-button-container add-button-container-{{item}}"
-                            data-ng-class="{
-                                'active': addedType.value == item
-                            }">
-                        </div>
+                <div class="col-md-6 col-lg-4 center add-button-item"
+                     data-ng-repeat="item in items.value"
+                     data-ng-class="{
+                        'active': addedType.value == item
+                     }">
 
-                        <a class="btn add-button" data-ng-class="{
-                            'active' : addedType.value == item
-                        }">{{ 'add_' + item | translate}}</a>
+                    <div class="form-group add-button-container add-button-container-{{item}}"
+                        data-ng-class="{
+                            'active': addedType.value == item
+                        }">
                     </div>
+
+                    <a class="btn add-button" data-ng-class="{
+                        'active' : addedType.value == item
+                    }">{{ 'add_' + item | translate}}</a>
                 </div>
+            </div>
 
-                <br/>
+            <br/>
 
-                <div class="block" data-ng-include="'app/cloud/project/storage/storage-add/storage/add-storage.html'"></div>
+            <div class="block" data-ng-include="'app/cloud/project/storage/storage-add/storage/add-storage.html'"></div>
```

Comment:
What does `.block` do ? `div` is already a block-level element

Reply:
https://github.com/ovh-ux/ovh-common-style/blob/master/less/helpers.less#L45

The `ovh-common-style` dependency has to be removed since it's in deprecated one! It should be addressed in a separate pull request.